src/content/ui-api/management-api/authentication/index.md

```diff
@@ -0,0 +1,117 @@
+---
+linkTitle: Authentication
+title: Authentication for the Management API
+description: How to authenticate for the management API as a user using single sign-on, plus some information for admins regarding how to set up single sign-on with your own identity provider.
+weight: 20
+menu:
+  main:
+    identifier: uiapi-managementapi-authentication
+    parent: uiapi-managementapi
+last_review_date: 2021-04-23
+user_questions:
+  - How can I authenticate as a user for the management API?
+  - How can I inspect the ID token issued for authenticating with the management API?
+  - What requirements are there to use single sign-on with the management API?
+  - How ca I refresh my groups memberships for management API authorization?
+---
+
+# Authentication for the Management API
+
+Here we explain how you'll authenticate against the management API as a user. For admins, we provide some information regarding the requirements for the initial [single sign-on setup](#sso-requirements) and as some [technical details](#technical-details) you might want to be interested in.
+
+## Authenticating as a user {#user-auth}
+
+As a user of the management API for any given installation, you need
+
+- A **user account** in the customer's identity provider (single sign-on).
+- The management API **endpoint URL** of the installation. Alternatively, the web user interface URL.
+
+### Using kubectl gs login
+
+Assuming that you want to work with the API using `kubectl`, the easiest method to authenticate is using the [Giant Swarm plug-in]({{< relref "/ui-api/kubectl-gs/_index.md" >}}) named `gs`. It can be [installed]({{< relref "/ui-api/kubectl-gs/installation/index.md" >}}) easily using `krew` and is recommended for all users of the management API.
```

Comment:
```suggestion
Assuming that you want to work with the API using `kubectl`, the easiest method to authenticate is by using the [Giant Swarm plug-in]({{< relref "/ui-api/kubectl-gs/_index.md" >}}) named `gs`. It can be [installed]({{< relref "/ui-api/kubectl-gs/installation/index.md" >}}) easily using `krew` and is recommended for all users of the management API.
```